be/src/vec/exec/scan/scanner_context.cpp

```diff
@@ -290,10 +290,12 @@ std::string ScannerContext::debug_string() {
 
 void ScannerContext::reschedule_scanner_ctx() {
```

Comment:
this function only call in [pip_scanner_context.h](https://github.com/apache/doris/pull/18306/files#diff-bc5230ea4bf2de1fb6721fd4231ed5a10a84347015b3650bb20a53b6657c1e83) should not in scanner_context